pkg/controllers/vdb/podfacts.go

```diff
@@ -399,6 +410,8 @@ func (p *PodFacts) genGatherScript(vdb *vapi.VerticaDB, pf *PodFact) string {
 		df --block-size=1 --output=avail %s | tail -1
 		echo -n 'agentRunning: '
 		/opt/vertica/sbin/vertica_agent status | grep --quiet "running" && echo true || echo false
+		echo -n 'hasAgentKeys: '
```

Comment:
Suggest a different name:
```suggestion
		echo -n 'imageHasAgentKeys: '
```

Reply:
Yes I really need to improve my naming skills!